src/Analyzers/CSharp/Tests/UseObjectInitializer/UseObjectInitializerTests.cs

```diff
@@ -1041,5 +1041,89 @@ class MyClass
                 }
                 """, OutputKind.ConsoleApplication);
         }
+
+        [Theory]
+        [InlineData(8.0)]
+        [InlineData(9.0)]
+        [WorkItem("https://github.com/dotnet/roslyn/issues/72094")]
+        public async Task TestWithConflictingSeverityConfigurationEntries(double analysisLevel)
```

Comment:
This is the unit test that mimics the behavior pointed out in #72094. Verified that this test fails for analysisLevel `8.0` prior to the product change